plugins/UM3NetworkPrinting/resources/qml/PrintJobInfoBlock.qml

```diff
@@ -2,13 +2,119 @@ import QtQuick 2.2
 import QtQuick.Controls 2.0
 import QtQuick.Controls.Styles 1.4
 import QtGraphicalEffects 1.0
+import QtQuick.Layouts 1.1
+import QtQuick.Dialogs 1.1
 
 import UM 1.3 as UM
 
 
 Item
 {
     id: base
+
+    function haveAlert() {
+        return printJob.configurationChanges.length !== 0;
```

Comment:
JavaScript calls are slower than direct assignments in properties.

Reply:
Also, shouldn't the config changes just load an item if there is a config change? This is what loaders are intended to do and it also ensures that items that are not needed don't get created.

Reply:
This is already being fixed in my branch from this branch where I'm rewriting the Print Info Block from scratch to be much cleaner. Don't spend more time reviewing this for now.